README.md

```diff
@@ -126,6 +124,47 @@ df2 = @byrow df begin
 end
 ```
 
+## Working with column names programmatically with `cols`
+
+DataFramesMeta provides the special syntax, `cols`, for referring to 
+columns in a DataFrame via a `Symbol`, `String`, or column position. 
```

Comment:
maybe replace `DataFrame` everywhere in the docs by "data frame" as you do not require a `DataFrame` but just `AbstractDataFrame` - right?

Reply:
same with `String` better write "string" as any string is allowed.

Reply:
Also it should be mentioned that `cols` allows passing a variable holding the index, right?

Reply:
added.